telegram/ext/filters.py

```diff
@@ -2075,6 +2075,18 @@ def filter(self, message: Message) -> bool:
                 and any(message.from_user.language_code.startswith(x) for x in self.lang)
             )
 
+    class _Attachment(MessageFilter):
+        name = 'Filters.attachment'
+
+        def filter(self, message: Message) -> bool:
+            return bool(message.effective_attachment)
+
+    attachment = _Attachment()
+    """Messages where :meth:`telegram.Message.effective_attachment` is not None.
+
+        See the documentation of that attribute when that is the case.
+        .. versionadded:: 13.6"""
```

Comment:
```suggestion
        See the documentation of that attribute for more details?
        
        .. versionadded:: 13.6
        """
```
IMHO we can just skip the first sentence. We don't have something like that for the other filters either
IIRC the new line before ..versionadded is necessary for it to render properly …

Reply:
> > IMHO we can just skip the first sentence
> 
> Sure. I just wanted to point out that that is the case, since I feel like thats less obvious then other cases

your call

Reply:
```suggestion

        .. versionadded:: 13.6"""
```